### DISCUSSION TITLE
PA2: Which test case should we pass?

### DISCUSSION THREAD

**Discussion by DHKim0428** [ANLAB-KAIST/KENSv3#55] [Project Q&A 2. Transport Layer (TCP)] [answered]
In kens' CMAKE list, there are many test cases like kens-part1-unreliable. 
Should we also pass the unreliable test cases?

**Comment by Nayoung-Oh** [comment, marked as answer]
(I am a student, too) After fetching the current master branch of this git repository, I can find the test cases are updated to have names like "test-kens-bind". However, I am not sure which one is directly related to this PA2, especially difference between 'unreliable' and the others.

**Reply by Pusnow** [reply]
You only need to pass reliable test cases.

**Reply by Nayoung-Oh** [reply]
So, does it mean that we need to pass "test-kens-bind", "test-kens-close", "test-kens-handshake", "test-kens-open" only to get a full point or should we pass "test-kens-transfer" and "test-kens-congestion" too? Thanks in advance!!

**Reply by Pusnow** [reply]
PA2: "test-kens-bind", "test-kens-close", "test-kens-handshake", "test-kens-open"
PA3: "test-transfer", "test-kens-bind-unreliable", "test-kens-close-unreliable", "test-kens-handshake-unreliable", "test-kens-open-unreliable", "test-transfer-unreliable"
